doc/add-mutated-by.md

```diff
@@ -0,0 +1,98 @@
+@function can-reflect-dependencies.addMutatedBy addMutatedBy
+@parent can-reflect-dependencies
+
+@description Register (one to many) mutation dependencies.
+
+@signature `.addMutatedBy(observable, [key], observableOrDependencyRecord)`
+
+Register a mutation dependency between an observable and optional `key`, to one or
+many observables.
+
+Mutation dependencies happen when an observable sets another observable's value,
+the following example creates two [simple observable](can-simple-observable.html)
```

Comment:
`[simple observable](can-simple-observable.html)` should probably be `[can-simple-observable simple observable]`